core/input/input_event.cpp

```diff
@@ -1717,6 +1717,18 @@ int InputEventMIDI::get_controller_value() const {
 	return controller_value;
 }
 
+void InputEventMIDI::open_inputs() {
+	OS::get_singleton()->open_midi_inputs();
```

Comment:
```suggestion
	ERR_FAIL_NULL(OS::get_singleton());
	OS::get_singleton()->open_midi_inputs();
```
Or
```suggestion
	if (OS::get_singleton()) {
		OS::get_singleton()->open_midi_inputs();
	}
```
I believe that's what's breaking the test project but unsure